pkg/cluster/cluster.go

```diff
@@ -132,11 +132,16 @@ func new(config Config, s *spec.ClusterSpec, stopC <-chan struct{}, wg *sync.Wai
 func (c *Cluster) prepareBackupAndRestore() error {
 	backup, restore := c.spec.Backup, c.spec.Restore
 
+	backupUsePV := backup.StorageType == spec.BackupStorageTypePersistentVolume || backup.StorageType == spec.BackupStorageTypeDefault
```

Comment:
we should use a switch statement here. s3 and pv should go completely different path.

```go
switch {
    s3:
    pv: preparePV
}

```